theforeman.org/pipelines/release/foreman_packaging_deb_release.groovy

```diff
@@ -0,0 +1,94 @@
+def packages_to_build
+def build_steps
+
+pipeline {
+    agent { label 'debian' }
+
+    options {
+        timestamps()
+        timeout(time: 4, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+                script {
+                    foreman_branch = foreman_version == 'nightly' ? "deb/develop" : "deb/${foreman_version}"
+                }
+
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: "*/${foreman_branch}"]],
+                    extensions: [[$class: 'CleanCheckout']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/foreman-packaging']
+                    ]
+                ])
+
+            }
+        }
+
+        stage('Find packages') {
+            steps {
+                copyArtifacts(projectName: env.JOB_NAME, optional: true)
+
+                script {
+
+                    if (fileExists('commit')) {
+
+                        commit = readFile(file: 'commit').trim()
+                        packages_to_build = find_changed_debs("${commit}..HEAD")
+                        build_steps = build_deb_package_steps(packages_to_build, foreman_version, 'theforeman', false)
+
+                    } else {
+
+                        build_steps
```

Comment:
```suggestion
                        build_steps = []
```

Reply:
or whatever is the correct syntax, to make the other steps be noop. probably `build_steps = ['setup_sources': [], 'executes': [], 'rsync_packages': []]` or so?